rails/empty-arrays-in-json-request.md

```diff
@@ -0,0 +1,66 @@
+Deep Munging
```

Comment:
There is a TIL that gives a high-level overview of `deep_munge` https://github.com/thoughtbot/til/blob/master/rails/deep_munge.md